components/hatch.py

```diff
@@ -13,13 +13,16 @@ class Hatch:
     hatch_left_puncher: wpilib.Solenoid
     hatch_right_puncher: wpilib.Solenoid
 
-    top_limit_switch: wpilib.DigitalInput
     left_limit_switch: wpilib.DigitalInput
     right_limit_switch: wpilib.DigitalInput
 
+    def setup(self):
+        wpilib.SmartDashboard.putData("hatch_left_switch", self.left_limit_switch)
+        wpilib.SmartDashboard.putData("hatch_right_switch", self.right_limit_switch)
+
     def on_enable(self):
         self._punch_on = False
-        self.has_hatch = True
+        self.has_hatch = False
```

Comment:
If we're still using this, this is probably a bad idea, as this would reset whether we have a hatch or not at the beginning of teleop.

If we're not still using this... why have it in the first place?